_IBC/agoric-archway.json

```diff
@@ -23,9 +23,32 @@
       "ordering": "unordered",
       "version": "ics20-1",
       "tags": {
-        "status": "live",
-        "preferred": true
+        "status": "live"
       }
     }
+  ],
+  "operators": [
```

Comment:
please remove the operators property from any files you've added it to. This isn't place the place to add that (at least not atm)